packages/api/src/Migrations/Version20190524102901.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Shopsys\ApiBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20190524102901 extends AbstractMigration
```

Comment:
Also, where are these tables used? Should it be placed in Api bundle?  I don't see usages here

Reply:
Imagine current situation
* you have SSFW v7
* we release v8 with the new API bundle

What do you have to do to use the API?
* add bundle to AppKernel
* add routes

And....
* If the migrations are here, that's more or less all
* If the migrations are in project-base, you have to copy them
  * also having migrations in project-base (where they are actually used) would complicate adding any further migrations - need for upgrade instructions

What do you think after this analysis? :-)

Reply:
I understand it. Seems like a compromise you had to make and I'm not telling it's wrong.
What if you don't want to use API? The migrations will still be done even if you choose to not apply the upgrade notes? If so, it is right?

I'm afraid of some unforeseen consequences of this. Will it complicate extendibility? What if we add some field (and this will be done in project-base, right?) and a project will need a different solution? Instead of 'I will do this differently' it has to be 'And don't forget to skip migration'.

I know skipping a migration is a  common concept in SSFW, but for a code used only in project-base it seems a little strange to me.

But both solutions look like you need to make things harder for someone. But I'm ok if it stays in package, but please be explicit about it in upgrade notes or documentation then. 

Please take into account a comment from henzigo, I think his points are valid and should be considered.